### PR TITLE
Floating User Menu Design Update (Alpha)

### PR DESCRIPTION
**Pull request by TomaszOlek**

### CODE REVIEW

**Comment by TomaszOlek** [comment]
I have used `theme.colors.textPrimary` In most places because I couldn't find a color in the theme that would work well

File: packages/utils/src/components/FloatingUserMenu.js [hunk 1]
```diff
@@ -163,40 +255,6 @@ const TextDiv = styled.div`
...
-  right: ${props => (props.isOpen ? "143px" : "9px")};
-  padding: 3.5px 5.5px;
-  border-radius: 50%;
-  background: red;
-  font-size: 10px;
-  line-height: 10px;
-  color: white;
-  font-weight: 700;
-`;
-
```
Comment (TomaszOlek) [inline packages/utils/src/components/FloatingUserMenu.js:199]: I have moved all styles to one place for better clearance of the code

File: packages/utils/src/components/FloatingUserMenu.js [hunk 2]
```diff
@@ -447,92 +504,88 @@ const UserMenuContextProvider = ({
                   )}
                 </React.Fragment>
               )}
+              {/* floating menu */}
```
Comment (TomaszOlek) [inline packages/utils/src/components/FloatingUserMenu.js:507]: Have added some comments so it is easier to find a certain thing when everything is collapsed

**Comment by hamzahalilovic** [comment]
Included in [#225](https://github.com/prifina/app-desktop/pull/225)

**Pull request closed without merging**